crates/starknet-server/src/api/json_rpc/spec_reader/mod.rs

```diff
@@ -170,7 +170,7 @@ mod tests {
     #[test]
     fn test_spec_methods() {
         let specs =
-            Spec::load_from_dir(concat!(env!("CARGO_MANIFEST_DIR"), "/test_data/spec/0.5.0"));
```

Comment:
so if we are changing 0.5.0 to 0.5.1 shouldn't we remove 0.5.0 folder?

Reply:
removed in [9aba1fc](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/257/commits/9aba1fc153a0e7fa6129a6abd75abb2960769c7b)